build/gulp.package.js

```diff
@@ -67,6 +67,7 @@ const zipArchive = async ({ osName, binding, tempBin, binaryName }) => {
   archive.directory(`${basePath}/binaries/${osName}/bin`, 'bin')
   archive.directory(`build/native-extensions/${binding}`, `bindings/${binding}`)
   archive.file(`${basePath}/binaries/${tempBin}`, { name: binaryName })
+  archive.file('build/sorry.macos.txt', { name: 'sorry.txt' })
```

Comment:
Won't the sorry file be added to every binaries (win and linux) that way?

```suggestion
  if (osName === 'darwin') {
    archive.file('build/sorry.macos.txt', { name: 'sorry.txt' })
  }
```